gubernator/static/build.js

```diff
@@ -149,15 +158,32 @@ function fix_escape_codes() {
 	}
 }
 
+/* Remove unicode sequences caused by colorized output in junit.xml */
+function remove_unicode_escape_codes() {
+	var errors = document.querySelectorAll('pre.error')
+	for (var i = 0; i < errors.length; i++) {
+		var orig = errors[i].innerHTML
+		var newer = orig.replace(/�\[\d+m/g, "")
```

Comment:
Unfortunately that's not the character that is actually in the innerHTML - it's the unrecognized char for now.